lib/context.d.ts

```diff
@@ -123,9 +123,9 @@ export interface StandardBindingContextTransforms {
 	hidden: BindingContextIdentityTransform<boolean>
 	html: BindingContextIdentityTransform<string>
 	class: BindingContextIdentityTransform<string>
-	css: BindingContextIdentityTransform<string | Record<string, boolean>>
-	style: BindingContextIdentityTransform<Record<string, string>>
-	attr: BindingContextIdentityTransform<Record<string, string>>
+	css: BindingContextIdentityTransform<string | Record<string, MaybeReadonlyObservable<boolean>>>
+	style: BindingContextIdentityTransform<Record<string, MaybeReadonlyObservable<string | number>>>
```

Comment:
Using numbers as the value in the 'style' binding handler is not supported.

```suggestion
	style: BindingContextIdentityTransform<Record<string, MaybeReadonlyObservable<string>>>
```